src/puppetlabs/puppetdb/config.clj

```diff
@@ -338,6 +336,13 @@
         (println "`ssl-protocols` contains SSLv3, a protocol with known vulnerabilities and should be removed from the `ssl-protocols` list"))))
   config-data)
 
+(defn warn-url-prefix-deprecation
```

Comment:
I think you can use [this](https://github.com/puppetlabs/puppetdb/blob/master/src/puppetlabs/puppetdb/utils.clj#L31) instead